src/components/Common/FilePreviewDialog.tsx

```diff
@@ -428,115 +431,132 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
-                      >
-                        {button[1] && (
-                          <CareIcon
-                            icon={button[1] as IconName}
-                            className="mr-2 text-lg"
-                          />
-                        )}
-                        {button[0] as string}
-                      </Button>
-                    ))}
+                    <Button
+                      variant="ghost"
+                      onClick={handleZoomOut}
+                      disabled={scale <= 0.5}
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                    >
+                      <CareIcon
+                        icon={"l-search-minus" as IconName}
+                        className="mr-2 text-lg"
+                      />
+                    </Button>
+
+                    <Button
+                      variant="ghost"
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                      disabled
+                    >
+                      {`${Math.round(scale * 100)}%`}
+                    </Button>
+
+                    <Button
+                      variant="ghost"
+                      onClick={handleZoomIn}
+                      disabled={scale >= 2}
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                    >
+                      <CareIcon
+                        icon={"l-search-plus" as IconName}
+                        className="mr-2 text-lg"
+                      />
+                    </Button>
+                  </>
+                )}
+              </div>
+              <div className="flex justify-center space-x-2">
+                {file_state.isImage && (
+                  <>
+                    <Button
+                      variant="ghost"
+                      onClick={() => handleRotate(90)}
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                    >
+                      <CareIcon
+                        icon={"l-corner-up-right" as IconName}
+                        className="text-lg size-4"
+                      />
+                      {t("rotate")}
+                    </Button>
+                  </>
+                )}
+                {file_state.extension === "pdf" && (
+                  <>
+                    <Button
+                      variant="ghost"
+                      onClick={() => setPage((prev) => prev - 1)}
+                      disabled={page === 1}
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                    >
+                      <CareIcon
+                        icon={"l-arrow-left" as IconName}
```

Comment:
```suggestion
                        icon="l-angle-left"
```

Let's use different set of icons to distinguish page navigation from file navigation

Reply:
Let's keep/add text for just these two. Unlike other buttons, these ones aren't as clear and could be misunderstood as navigating to previous/next file.

So let's add `Previous Page` and `Next Page` text next to the buttons.

Reply:
Will add the text and change the icons in next commit and also tick the boxes from checklist